techniques/css/C30.html

```diff
@@ -12,106 +11,74 @@
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>Any technology that supports CSS. </p>
-            <p>This technique relates to:</p>
+            <h2>適用 (対象)</h2>
+            <p>CSS をサポートするすべてのウェブコンテンツ技術 </p>
+            <p>これは、次の達成基準に関する達成方法である:</p>
             <ul>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/images-of-text">Success Criterion 1.4.5: Images of Text</a> (Sufficient)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/images-of-text">達成基準 1.4.5: 文字画像</a> (十分な達成方法)
                </li>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/images-of-text-no-exception">Success Criterion 1.4.9: Images of Text (No Exception)</a> (Sufficient)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/images-of-text-no-exception">達成基準 1.4.9: 文字画像 (例外なし)</a> (十分な達成方法)
                </li>
             </ul>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to demonstrate how CSS can be used to replace structured
-               HTML text with images of text in a way that makes it possible for users to view content
-               according to their preferences. To use this technique, an author starts by creating
-               an HTML page that uses semantic elements to mark up the structure of the page. The
-               author then designs two or more stylesheets for that page. One stylesheet presents
-               the HTML text as text and the second uses CSS features to replace some of the HTML
-               text with images of text. Finally, through the use of server-side or client-side scripting,
-               the author provides a control that allows the user to switch between the available
-               views. 
+            <h2>解説</h2>
+            <p>この達成方法の目的は、利用者が彼らの好みに従ってコンテンツを見ることができるような方法で構造化された HTML テキストを画像化されたテキストに置き換えるための CSS の用い方を実践することである。この達成方法を用いるには、コンテンツ制作者はページの構造をマークアップするためにセマンティックな要素を用いる HTML ページを作ることから始める。次にコンテンツ制作者はページに対して 2 またはそれ以上のスタイルシートを作成する。一つめのスタイルシートは HTML テキストをテキストとして表現し、二つめはいくつかの HTML テキストをテキストの画像に置き換えるために CSS の機能を用いている。最後に、server-side 又は client-side scripting の使用を通して、コンテンツ制作者は利用者が利用するビューを切り替えることができるコントロールを提供する。
```

Comment:
images of textが文字画像になっていなかったので、それとあわせて少し手を加えました。

```suggestion
            <p>この達成方法の目的は、利用者が好みに従ってコンテンツを表示できるように、構造化された HTML テキストを文字画像に置き換える CSS の用い方を示すことである。この達成方法を用いるには、コンテンツ制作者はページの構造をマークアップするためにセマンティックな要素を用いる HTML ページを作ることから始める。次にコンテンツ制作者はページに対して二つ以上のスタイルシートを作成する。一つめのスタイルシートは HTML テキストをテキストとして表現し、二つめはいくつかの HTML テキストを文字画像に置き換えるために CSS の機能を用いている。最後に、サーバーサイド又はクライアントサイドスクリプトの使用を通して、コンテンツ制作者は利用者が利用するビューを切り替えることができるコントロールを提供する。
```